src/node.rs

```diff
@@ -73,6 +73,30 @@ use std::{fmt, fs};
 use tokio::runtime::Handle;
 use tokio::task::JoinHandle;
 
+#[derive(Serialize, Debug)]
+pub struct LocalInvoice {
+    hash: String,
+    currency: String,
+    amount: u64,
+    description: String,
+    expiry: u64,
+}
+
+impl std::convert::From<Invoice> for LocalInvoice {
+    fn from(invoice: Invoice) -> Self {
+        Self {
+            hash: invoice.payment_hash().to_string(),
```

Comment:
```suggestion
            payment_hash: invoice.payment_hash().to_string(),
```
It might be better to keep `payment_hash` to aovid confusion with possible `description_hash`.